File: Difficulty/Easy/[171] Excel Sheet Column Number/solution.py
class Solution:
    def titleToNumber(self, s: str) -> int:
        length, res = len(s) - 1, 0
        for i in range(len(s)):
            res += ((ord(s[i]) - ord('A')) + 1) * math.pow(26, length - i)
        return int(res)

File: Difficulty/Easy/[171] Excel Sheet Column Number/solution.cpp
/*
 * @lc app=leetcode id=171 lang=cpp
 *
 * [171] Excel Sheet Column Number
 */

// @lc code=start
class Solution {
public:
    int titleToNumber(string s) {
        int length = s.size() - 1;
        double res = 0;
        for (int i = 0; i < s.size(); i++)
            res += ((s[i] - 'A') + 1) * pow(26, length - i);   
        return int(res);
    }
};
// @lc code=end

